lib/katello/tasks/upgrades/3.6/import_backend_consumer_attributes.rake

```diff
@@ -0,0 +1,23 @@
+namespace :katello do
+  namespace :upgrades do
+    namespace '3.6' do
+      desc "Import product content from Candlepin to improve API performance and enhance searching"
+      task :import_backend_consumer_attributes => %w(environment check_ping) do
+        User.current = User.anonymous_admin
+        Organization.all.each do |org|
+          consumers = ::Katello::Resources::Candlepin::Consumer.get(:owner => org.label,
+                                                                     :include_only => %w(uuid installedProducts.productId
+                                                                                         installedProducts.productName installedProducts.version
+                                                                                         installedProducts.arch))
+          consumers.each do |consumer|
+            facet = Katello::Host::SubscriptionFacet.find_by(:uuid => consumer['uuid'])
+            if facet
```

Comment:
Are rake tasks able to log to production.log? If this takes awhile to run, it would be useful to print a line every 100th host or something that says `host #{done_so_far} of #{total}`

Reply:
updated, i added it ever 25th host.